index.html

```diff
@@ -356,7 +356,7 @@ <h1 class="title">Promise.prototype.finally</h1>
 		</emu-alg>
 	</emu-clause>
 
-	<!-- es6num="25.4.1.8" -->
+	<!-- es6num="25.4.4.5" -->
```

Comment:
@bterlson this seems to have changed between when I initially wrote this PR and now. Was this intentional? I thought "es6num" values were supposed to stay constant forever?